test/integration/suites/node-re-attestation/02-start-agent

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+source ./common
+
+log-debug "bootstrapping agent..."
+docker-compose exec -T spire-server \
+    /opt/spire/bin/spire-server bundle show > conf/agent/bootstrap.crt
+
+log-info "generating join token..."
+TOKEN=$(docker-compose exec -T spire-server \
+    /opt/spire/bin/spire-server token generate -spiffeID spiffe://domain.test/node | awk '{print $2}' | tr -d '\r')
```

Comment:
nit: you can use json output to make it easier

Reply:
761d9ec16eb6929f5ea21d2fdf0c523c6acb9bec